pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperKitCreationService.java

```diff
@@ -24,39 +24,33 @@
 @Slf4j
 public class NonPepperKitCreationService {
     public static final String JUNIPER = "JUNIPER";
-    public static final String JUNIPER_UNDERSCORE = "JUNIPER_";
 
     //These are the Error Strings that are expected by Juniper
-    public static final String ADDRESS_VALIDATION_ERROR = "UNABLE_TO_VERIFY_ADDRESS";
-    public static final String UNKNOWN_KIT_TYPE = "UNKNOWN_KIT_TYPE";
-    public static final String UNKNOWN_STUDY = "UNKNOWN_STUDY";
-    public static final String MISSING_JUNIPER_KIT_ID = "MISSING_JUNIPER_KIT_ID";
-    public static final String MISSING_JUNIPER_PARTICIPANT_ID = "MISSING_JUNIPER_PARTICIPANT_ID";
-    public static final String DSM_ERROR = "DSM_ERROR_SOMETHING_WENT_WRONG";
+
 
     public KitResponse createNonPepperKit(JuniperKitRequest juniperKitRequest, String studyGuid, String kitTypeName) {
         if (StringUtils.isBlank(juniperKitRequest.getJuniperParticipantID())) {
-            return new KitResponse(MISSING_JUNIPER_PARTICIPANT_ID, juniperKitRequest.getJuniperKitId(),
+            return new KitResponseError(KitResponse.MISSING_JUNIPER_PARTICIPANT_ID, juniperKitRequest.getJuniperKitId(),
```

Comment:
This would typically be a 400 error, but it looks like it probably returns a 200?
Likewise for the following checks.

Reply:
In `JuniperShipKitRoute` There is a check for the response of this method
```
if (!(kitResponse instanceof KitResponseError)) {
            response.status(200);
        } else {
            response.status(400);
        }
```